prime-angular-frontend/src/app/modules/site-registration/pages/care-setting/care-setting.component.ts

```diff
@@ -85,7 +85,7 @@ export class CareSettingComponent implements OnInit, IPage, IFormPage {
         actionText: 'Continue'
       };
       const update$ = this.siteResource.updateSite(payload);
-      const request$ = (payload.siteVendors[0].vendorCode === VendorEnum.CARECONNECT && payload.remoteUsers.length)
+      const request$ = (payload.siteVendors[0].vendorCode === VendorEnum.CARECONNECT && payload.remoteUsers?.length)
```

Comment:
This shouldn't be possible since the default value for remoteUsers is an empty array